4717_Sprint/04 - xander/OCP-CH6/src/com/company/Demo.java

```diff
@@ -0,0 +1,80 @@
+package com.company;
+
+
+import java.io.*;
+import java.time.DateTimeException;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+
+public class Demo {
+
+    public static void main(String[] args) throws IOException {
+
+        File file;
+        try{
+          file = Demo.createFile();
+        }catch(IOException e) {
+            System.out.println(e.getMessage());
+            throw new RuntimeException(e);
+        }
+
+        String[] lines = {"Multiple","things","to","write","to","file","yo"};
+        Demo.writeStringToFile("Single",file);
+        Demo.writeStringToFile(lines,file);
+        Demo.readFromFile(file).forEach(System.out::println);
+
+        try {
+            int z = divideBy(10,0);
+            LocalDateTime time = LocalDateTime.parse("test");
+        } catch (DateTimeException | ArithmeticException e) {
```

Comment:
Switch the order of the exceptions, so the first one defined on the catch is the first one happening in the try